applications/common/backend/src/main/resources/libs/kyanite/common/components/codesnippet/dialog/.content.json

```diff
@@ -60,6 +60,25 @@
         "sling:resourceType": "wcm/dialogs/components/textarea",
         "name": "code",
         "label": "Code"
+      },
+      "isExpandingOn": {
+        "sling:resourceType": "wcm/dialogs/components/toggle",
+        "name": "isExpandingOn",
+        "label": "Expandable snippets"
+      },
+      "expandContainer": {
+				"sling:resourceType": "wcm/dialogs/components/container",
```

Comment:
Looks like tabs got here instead of spaces;)

Reply:
it looks good in the project, maybe github doesnt see difference when white spaces changed

Reply:
That's probably because IDE displays tabs using width=4, whereas here tab is displayed with size of 8:
![image](https://github.com/websight-io/kyanite/assets/165791712/3cbb66a3-12c5-46b7-ba3c-5b00560069e1)

But that's only a minor issue